### ISSUE TITLE
IndexSizeError in Link example

### ISSUE THREAD

**Issue by jaredpalmer** [ianstormtaylor/slate#76]
Tried out the `linkify` example. Having issues when I try to set multiple links.

```
leaf.js?1f99:141 Uncaught IndexSizeError: Failed to execute 'setStart' on 'Range': 
The offset 7 is larger than or equal to the node's length (4).
``` 

This project is epic btw.

**Comment by ianstormtaylor** [comment]
Thanks! Interesting, checking it out. Looks like it only happens when the second link has less text in it that the first. Looking into this now!

**Comment by ianstormtaylor** [comment]
Fixed in `0.1.8`! Thanks for catching that.
